### PR TITLE
Params refactored

### PR DESCRIPTION
**Pull request by FacundoPucheta**
- Add latitude and longitude to parameters.
- All parameters are parsed properly to perform better at charts.

### CODE REVIEW

File: src/handlers/getExceedanceProbabilityHandler.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 const getExceedanceProbability = require("../controllers/getExceedanceProbability");
 
 const getExceedanceProbabilityHandler = async (req, res) => {
-  const { location, period } = req.query;
+  let { lat, long, location, period } = req.query;
 
   try {
-    if (!location || !period) throw { status: 400, message: "Missed location or period." };
+    if (!lat || !long || !location || !period) throw { status: 400, message: "Missed location or period." };
+    if(period.slice(period.length-1) == 0) period = parseInt(period);
```
Comment (SebaSatelier) [inline src/handlers/getExceedanceProbabilityHandler.js]: maybe let some spaces here, betwen this lines?

File: src/helpers/asce.js [hunk 2]
```diff
@@ -39,12 +39,12 @@ const faAsce = (Ss, tp) => {
   return F[tp][5];
 };
 
-const asce = async (location, soilType) => {
+const asce = async (lat, long, soilType) => {
 
-  let prob = await probabilities(location, 0.2);
+  let prob = await probabilities(lat, long, 0.2);
```
Comment (SebaSatelier) [inline src/helpers/asce.js]: in probabilities helper, don't you use location inside?

File: src/helpers/asce.js [hunk 3]
```diff
@@ -39,12 +39,12 @@ const faAsce = (Ss, tp) => {
...
 
-const asce = async (location, soilType) => {
+const asce = async (lat, long, soilType) => {
 
-  let prob = await probabilities(location, 0.2);
+  let prob = await probabilities(lat, long, 0.2);
   let Ss = interpolation(prob, 1.0 / 2475.0); //PGA1
 
-  let prob2 = await probabilities(location, 1.0);
+  let prob2 = await probabilities(lat, long, 1.0);
```
Comment (SebaSatelier) [inline src/helpers/asce.js]: in probabilities helper, don't you use location inside?

File: src/helpers/asce.js [hunk 4]
```diff
@@ -39,12 +39,12 @@ const faAsce = (Ss, tp) => {
   return F[tp][5];
 };
 
-const asce = async (location, soilType) => {
+const asce = async (lat, long, soilType) => {
```
Comment (SebaSatelier) [inline src/helpers/asce.js:43]: should receive location by parameter?

File: src/helpers/ibc.js [hunk 5]
```diff
@@ -37,11 +37,11 @@ const fa = (Ss, tp) => {
 	return F[tp][4];
 };
 
-const ibc = async (location, soilType) => {
-	let prob = await probabilities(location, 0.2);
+const ibc = async (lat, long, soilType) => {
```
Comment (SebaSatelier) [inline src/helpers/ibc.js]: should receive location by parameter?

File: src/helpers/ibc.js [hunk 6]
```diff
@@ -37,11 +37,11 @@ const fa = (Ss, tp) => {
...
+	let prob = await probabilities(lat, long, 0.2);
```
Comment (SebaSatelier) [inline src/helpers/ibc.js]: in probabilities helper, don't you use location inside?

File: src/helpers/ibc.js [hunk 7]
```diff
@@ -37,11 +37,11 @@ const fa = (Ss, tp) => {
...
 	let Ss = interpolation(prob, 1.0 / 2475.0); //PGA1
 
-	let prob2 = await probabilities(location, 1.0);
+	let prob2 = await probabilities(lat, long, 1.0);
```
Comment (SebaSatelier) [inline src/helpers/ibc.js]: in probabilities helper, don't you use location inside?

File: src/helpers/standardE30_2015Spec.js [hunk 8]
```diff
@@ -53,7 +50,7 @@ const standardE30_2015Spe = async (location, soilType) => {
         break;
     }
 
-    const prob = await probabilities(location, period);
+    const prob = await probabilities(lat, long, period);
```
Comment (SebaSatelier) [inline src/helpers/standardE30_2015Spec.js]: in probabilities helper, don't you use location inside?

File: src/helpers/standardE30_2015Spec.js [hunk 9]
```diff
@@ -1,18 +1,15 @@
-const { Location } = require("../db");
 const getPolygon = require("./getPolygon");
 const getDBCoordinates = require("./getDBCoordinates");
 const getDBPolygons = require("./getDBPolygons");
 const getPeriodArray = require("./getPeriodArray");
 const probabilities = require("./probabilities");
 const interpolation = require("./interpolation");
 
-const standardE30_2015Spe = async (location, soilType) => {
+const standardE30_2015Spe = async (lat, long, soilType) => {
```
Comment (SebaSatelier) [inline src/helpers/standardE30_2015Spec.js]: should receive location by parameter?

**Comment by SebaSatelier** [comment]
leave some comments to review

**Review by SebaSatelier — APPROVED** [review]
Looks good!

File: src/helpers/asce.js [hunk 10]
```diff
@@ -39,12 +39,12 @@ const faAsce = (Ss, tp) => {
   return F[tp][5];
 };
 
-const asce = async (location, soilType) => {
+const asce = async (lat, long, location, soilType) => {
```
Comment (manduinca) [inline src/helpers/asce.js:42]: what is the difference between lat, long and location?

Comment (FacundoPucheta) [reply src/helpers/asce.js:42]: Location has ID as its value (obtained from getLocation, when the component is mounted). We store this value in a global state, The purpose is to use "location" only for searching on zar's models(by id). 
As a result, **the request to DB to obtain latitude and longitude do not happen** (we pass latitude and longitude by parameters, wich are already stored in the global state)

**Review by manduinca — APPROVED** [review]

**Pull request merged**